src/ergw_aaa_radius.erl

```diff
@@ -39,10 +39,10 @@
     {normal, 1},
     {administrative, 6},
     {link_broken, 2},
-    {upf_failure, 2},
-    {session_error, 2},
+    {upf_failure, 9},
+    {remote_failure, 9},
```

Comment:
I'm still not convinced that code 9 `NAS Error` is correct. IMHO 2 `Lost Carrier` is more appropriate.

@mgumz what is you take on this? Do we have a document describing our default mapping (I don't care what customers might thing, we can configure a non standard mapping for them).